decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -167,6 +175,18 @@ namespace :decidim do
     def system!(command)
       system("cd #{rails_app_path} && #{command}") || abort("\n== Command #{command} failed ==")
     end
+
+    def tailwind_variables
+      # Get the list of Decidim gems installed and adds the current folder ('.')
+      content_directories = Dir.glob("#{Gem.loaded_specs["decidim"].full_gem_path}/decidim-*").push(".")
```

Comment:
If you are writing these paths to the static configuration file, you will face issues when you move the same instance to another environment, e.g. from dev to staging/production. Therefore, I would not statically write these to the configuration files as they depend on the environment.

Also, how are now modules added to the configuration paths? As I understood this is only run during `decidim:webpacker:install`.